packages/react/src/Pagination/Pagination.tsx

```diff
@@ -10,22 +10,22 @@ import type { ForwardedRef, HTMLAttributes } from 'react'
 import { Icon } from '../Icon/Icon'
 
 export type PaginationProps = {
-  /**
-   * The maximum amount of pages shown. This has a lower limit of 5
-   */
+  /** The maximum amount of pages shown. This has a lower limit of 5. */
   maxVisiblePages?: number
-  /**
-   * Callback triggered when interaction changes the page number.
-   */
+  /** The accessible name for the link to the next page. */
+  nextAriaLabel?: string
+  /** The label for the link to the next page. */
+  nextLabel?: string
```

Comment:
I get your point, but I would say having them start with the same word is enough coupling. Code reviews and accessibility testing should prevent diverging labels here.

Reply:
I was referring to the values not the prop names. 

I'm not so sure about the `nextLabel` object. How would you check?

```jsx
{ nextLabel?.aria }
```

Reply:
@dlnr There’s a misunderstanding here. Aram wants to ensure that a developer updates both the `nextLabel` and `nextAriaLabel` props if a textual change occurs. One might change the value of one of the two props but forget the other,  introducing inconsistent values. We’re not discussing the actual values or translations.

Grouping the two props in an object may decrease the chance of failure here. My point is that the props are already next to each other because they both start with `next`, so that an object doesn’t help much for discoverablity. I also dislike object props.

The same goes for the two `previous` props.

Reply:
Let me cut the knot here: the current approach for these props is sufficient. We can only go so far in coaching people what values to assign them.

Reply:
I did add a line to the Pagination guidelines for this – only in [an entirely different PR](https://github.com/Amsterdam/design-system/pull/1227/commits/b1f71ed523d202c8621483e1c9995beffe007def) 🤦🏼 😎